sound/soc/sof/sof-priv.h

```diff
@@ -527,6 +527,9 @@ int snd_sof_ipc_set_get_comp_data(struct snd_sof_ipc *ipc,
 				  enum sof_ipc_ctrl_cmd ctrl_cmd,
 				  bool send);
 
+/* DSP core enable/disable IPC */
+int snd_sof_ipc_core_enable(struct snd_sof_dev *sdev);
```

Comment:
I think Pierre already suggested that - would be good to also switch over to using this function in the same patch